pkg/executor/brie_test.go

```diff
@@ -140,3 +144,82 @@ func TestFetchShowBRIE(t *testing.T) {
+	defer failpoint.Disable("github.com/pingcap/tidb/pkg/executor/modifyStore")
+	err := os.WriteFile("/tmp/keyfile", []byte(strings.Repeat("A", 128)), 0644)
+
+	require.NoError(t, err)
+	stmt, err := p.ParseOneStmt("BACKUP TABLE `a` TO 'noop://' CHECKSUM_CONCURRENCY = 4 IGNORE_STATS = 1 COMPRESSION_LEVEL = 4 COMPRESSION_TYPE = 'lz4' ENCRYPTION_METHOD = 'aes256-ctr' ENCRYPTION_KEYFILE = '/tmp/keyfile'", "", "")
+	require.NoError(t, err)
+	plan, err := core.BuildLogicalPlanForTest(ctx, sctx, stmt, infoschema.MockInfoSchema([]*model.TableInfo{core.MockSignedTable(), core.MockUnsignedTable(), core.MockView()}))
+	require.NoError(t, err)
+	s, ok := stmt.(*ast.BRIEStmt)
+	require.True(t, ok)
+	require.True(t, s.Kind == ast.BRIEKindBackup)
+	for _, opt := range s.Options {
+		switch opt.Tp {
+		case ast.BRIEOptionChecksumConcurrency:
+			require.True(t, opt.UintValue == 4)
+		case ast.BRIEOptionCompressionLevel:
+			require.True(t, opt.UintValue == 4)
+		case ast.BRIEOptionIgnoreStats:
+			require.True(t, opt.UintValue == 1)
+		case ast.BRIEOptionCompression:
+			require.True(t, opt.StrValue == "lz4")
+		case ast.BRIEOptionEncryptionMethod:
+			require.True(t, opt.StrValue == "aes256-ctr")
+		case ast.BRIEOptionEncryptionKeyFile:
+			require.True(t, opt.StrValue == "/tmp/keyfile")
+		}
+	}
+	schema := plan.Schema()
+	exec := builder.buildBRIE(s, schema)
+	require.NoError(t, builder.err)
+	e, ok := exec.(*BRIEExec)
+	require.True(t, ok)
+	require.True(t, e.backupCfg.ChecksumConcurrency == 4)
+	require.True(t, e.backupCfg.CompressionLevel == 4)
+	require.True(t, e.backupCfg.IgnoreStats)
+	require.True(t, e.backupCfg.CompressionConfig.CompressionType == backuppb.CompressionType_LZ4)
+	require.True(t, e.backupCfg.CipherInfo.CipherType == encryptionpb.EncryptionMethod_AES256_CTR)
+	require.True(t, len(e.backupCfg.CipherInfo.CipherKey) > 0)
+
+	stmt, err = p.ParseOneStmt("RESTORE TABLE `a` FROM 'noop://' CHECKSUM_CONCURRENCY = 4 WAIT_TIFLASH_READY = 1 WITH_SYS_TABLE = 1 LOAD_STATS = 1", "", "")
+	require.NoError(t, err)
+	plan, err = core.BuildLogicalPlanForTest(ctx, sctx, stmt, infoschema.MockInfoSchema([]*model.TableInfo{core.MockSignedTable(), core.MockUnsignedTable(), core.MockView()}))
+	require.NoError(t, err)
+	s, ok = stmt.(*ast.BRIEStmt)
+	require.True(t, ok)
+	require.True(t, s.Kind == ast.BRIEKindRestore)
+	for _, opt := range s.Options {
+		switch opt.Tp {
+		case ast.BRIEOptionChecksumConcurrency:
+			require.True(t, opt.UintValue == 4)
+		case ast.BRIEOptionWaitTiflashReady:
+			require.True(t, opt.UintValue == 1)
+		case ast.BRIEOptionWithSysTable:
+			require.True(t, opt.UintValue == 1)
+		case ast.BRIEOptionLoadStats:
+			require.True(t, opt.UintValue == 1)
+		}
+	}
+	schema = plan.Schema()
+	exec = builder.buildBRIE(s, schema)
+	require.NoError(t, builder.err)
+	e, ok = exec.(*BRIEExec)
+	require.True(t, ok)
+	require.True(t, e.restoreCfg.ChecksumConcurrency == 4)
```

Comment:
Preferring use `require.Equal`, which will show the values when assertion fails.
```suggestion
	require.Equal(t, e.restoreCfg.ChecksumConcurrency, 4)
```